packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -55,8 +55,12 @@
   }
 
   .help-button {
-    margin: 15px;
+    padding: 20px;
+    position: fixed;
+    position: sticky;
```

Comment:
how are we defining postion twice? Sorry not super familar with sticky

Reply:
Edge/ie does not support sticky position. I was trying to apply CSS specific to edge/ie and set the position fixed for them. Talked to Mike and he suggested to set position: fixed above sticky. Tested it and from my understanding, if sticky is not supported it falls back to fixed.